compiler/jsgen.nim

```diff
@@ -1567,10 +1567,17 @@ proc genVarInit(p: PProc, v: PSym, n: PNode) =
   var
     a: TCompRes
     s: Rope
+    varCode: string
+    returnType: Rope
```

Comment:
This doesn't have to be a ``var``  and also not a ``let``, you can inline the ``~""``  expression.

Reply:
Yeah, I wanted to name it for clarity, because otherwise people might wonder why the ~"" is there

Reply:
template returnType(): untyped = ~""

Reply:
Ok, fixed in last commit